ui-components/src/global/mixins.scss

```diff
@@ -65,11 +65,12 @@
 @mixin filled-appearances() {
   &.is-primary {
     @apply bg-primary-dark;
-    @apply border-primary;
+    @apply border-primary-dark;
```

Comment:
these changes don't seem to be working for me? I'm still seeing a difference in the color on 
https://deploy-preview-1596--dev-bloom.netlify.app/?utm_source=github&utm_campaign=bot_dp
![image](https://user-images.githubusercontent.com/11825994/127707383-32562d1c-89c2-4a08-9488-5f4f9ac8046a.png)